content/applications/sales/subscriptions.rst

```diff
@@ -4,20 +4,22 @@
 Subscriptions
 =============
```

Comment:
Note @meng-odoo
Can clean up 2nd-person on this doc + give the content another pass on a new PR.